### ISSUE TITLE
Warn when leaving Edit Votes page with unsaved changes

### ISSUE THREAD

**Issue by MichaelNMaggs** [hatnote/montage#137]
It's possible to leave the Edit Votes page without saving changes. If a jury member does that, any changes are silently lost. 

There should be a warning if the user attempts to leave the page without saving (especially as the main voting page records each vote as it's made without a separate save step).

**Comment by mahmoud** [comment]
Good catch! Almost missed this on a recent edit for a round I'm judging, thankfully I was able to go back and find the vote, change it, and actually hit save. Thanks! Looks like Pawel's seen it, so here's hoping it's fixed soon :)
